Spanner/src/Interval.php

```diff
@@ -79,6 +79,8 @@ public static function parse(string $text): Interval
             throw new InvalidArgumentException('The given interval is empty.');
         }
 
+        $text = str_replace(',', '.', $text);
```

Comment:
Should we be doing this? I don't see any case of this in the test. Wouldn't it be best to just reject it as invalid and leave it to the customer to do this?

And if we do need to support this, there should be a test for it.

Reply:
`,` is not invalid. You can pass a `,` and is valid, but the returned value always returns with a dot.

Line 92 on `IntervalTest` has one with a comma.

Reply:
Ahh ok! I missed it. LGTM then.

please just add a comment saying something like `// Interval also accepts decimals delimited with a comma instead of a period`